src/Kdyby/Redis/DI/RedisExtension.php

```diff
@@ -68,11 +68,10 @@ public function loadConfiguration()
 		$builder = $this->getContainerBuilder();
 		$config = self::fixClientConfig($this->getConfig($this->defaults + $this->clientDefaults));
 
-		$this->buildClient(NULL, $config);
-
 		$builder->addDefinition($this->prefix('driver'))
-			->setClass(class_exists('Redis') ? 'Kdyby\Redis\Driver\PhpRedisDriver' : 'Kdyby\Redis\IRedisDriver')
-			->setFactory($this->prefix('@client') . '::getDriver');
+			->setClass(Kdyby\Redis\Driver\PhpRedisDriver::class);
```

Comment:
Return the class_exists. I think it's there to allow this extention to be installed on environment where Redis class is not available.